x-pack/plugins/security_solution/public/management/links.ts

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+import {
+  BLOCKLIST_PATH,
+  ENDPOINTS_PATH,
+  EVENT_FILTERS_PATH,
+  HOST_ISOLATION_EXCEPTIONS_PATH,
+  MANAGEMENT_PATH,
+  POLICIES_PATH,
+  SecurityPageName,
+  TRUSTED_APPS_PATH,
+} from '../../common/constants';
+import {
+  BLOCKLIST,
+  ENDPOINTS,
+  EVENT_FILTERS,
+  HOST_ISOLATION_EXCEPTIONS,
+  MANAGE,
+  POLICIES,
+  TRUSTED_APPLICATIONS,
+} from '../app/translations';
+import { FEATURE, LinkItem } from '../common/links/types';
+
+export const links: LinkItem = {
+  id: SecurityPageName.manageLanding,
+  title: MANAGE,
+  path: MANAGEMENT_PATH,
+  skipUrlState: true,
+  globalNavEnabled: false,
+  features: [FEATURE.general],
+  globalSearchKeywords: [
+    i18n.translate('xpack.securitySolution.search.manage', {
+      defaultMessage: 'Manage',
+    }),
+  ],
+  links: [
+    {
+      id: SecurityPageName.endpoints,
+      globalNavEnabled: true,
+      title: ENDPOINTS,
+      globalNavOrder: 9006,
+      path: ENDPOINTS_PATH,
+      skipUrlState: true,
+    },
+    {
+      id: SecurityPageName.policies,
+      title: POLICIES,
+      path: POLICIES_PATH,
+      skipUrlState: true,
+      experimentalKey: 'policyListEnabled',
+    },
+    {
+      id: SecurityPageName.trustedApps,
+      title: TRUSTED_APPLICATIONS,
+      path: TRUSTED_APPS_PATH,
+      skipUrlState: true,
+    },
+    {
+      id: SecurityPageName.eventFilters,
+      title: EVENT_FILTERS,
+      path: EVENT_FILTERS_PATH,
+      skipUrlState: true,
+    },
+    {
+      id: SecurityPageName.hostIsolationExceptions,
+      title: HOST_ISOLATION_EXCEPTIONS,
+      path: HOST_ISOLATION_EXCEPTIONS_PATH,
+      skipUrlState: true,
+    },
+    {
+      id: SecurityPageName.blocklist,
+      title: BLOCKLIST,
+      path: BLOCKLIST_PATH,
+      skipUrlState: true,
+    },
+  ],
+};
```

Comment:
Can we add ` as const` here?

Reply:
why?

Reply:
Because...
1. value is defined as a global module const (but we all know Object's are not really true const's)
2. Value can be mutated from anywhere in the app (just import and change its data
3. If opposed to using `as const` which gives us really only dev-time TS validations, then perhaps `Object.freeze()` it?

Reply:
Given the [additional context provided](https://github.com/elastic/kibana/pull/131157#discussion_r866117197) by @semd (thanks again for that), I now understand why this was deliberately made mutable.